magento/pwa-studio
# [PWA-612]: Fixed shipping method loading error.

## Description

Fixed a false negative in shipping method component.

The component has a check to show an error message is shipping methods array is empty, but the array can be empty if the query is still loading. That is a false negative.

Fixed it by adding the query loading state to the if check.

## Related Issue
Closes [PWA-612](https://jira.corp.magento.com/browse/PWA-612)

## Acceptance 
The component should not show any error message while loading.

### Verification Stakeholders
@supernova-at 
@jimbo 

### Verification Steps
1. Go to the checkout page with at least 1 item in cart.
2. Continue to step 2. Select a shipping method.
3. After the shipping method is set, refresh the page, there should not be any error message shown before the shipping method summary is shown.

## Screenshots / Screen Captures (if appropriate)

Before:
![image](https://user-images.githubusercontent.com/35203638/84314591-ec9fe000-ab2d-11ea-8ddb-d52bd44408dc.png)

After:
![image](https://user-images.githubusercontent.com/35203638/84314706-1d801500-ab2e-11ea-8263-e92d2e277bae.png)

## Checklist
